src/app/space/create/codebases/codebases-item/codebases-item.component.html

```diff
@@ -27,4 +27,11 @@
   <div class="list-pf-additional-content">
```

Comment:
This should be moved inside of the `<ng-template>` and change the class to `list-pf-description`. That will ensure that the content lays out and collapses properly.

There should only be one `<div>` of `'list-pf-additional-content'.